includes/wikia/tests/UserPasswordTest.php

```diff
@@ -34,7 +34,7 @@ public static function tearDownAfterClass() {
 	private function setupAndInjectServiceMocks() {
 		$this->heliosClientMock = $this->getMock( HeliosClient::class,
 			[ 'login', 'forceLogout', 'invalidateToken', 'register', 'info', 'generateToken',
-				'setPassword', 'validatePassword', 'deletePassword' ] );
+				'setPassword', 'validatePassword', 'deletePassword', 'requestPasswordReset' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "setPassword" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "deletePassword" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "validatePassword" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)